sdk/tests/bn/test.ts

```diff
@@ -108,6 +115,12 @@ describe('BigNum Tests', () => {
 		expect(val4.toNum().toFixed(3)).to.equal('0.025');
 		expect(val4.toPrecision(4)).to.equal('0.025');
 
+		expect(bnBaseAmount(0.001234).toPrecision(4)).to.equal('0.001234');
+
+		expect(bnBaseAmount(0.001004).toPrecision(4)).to.equal('0.001004');
+
+		expect(bnBaseAmount(0.001).toPrecision(4)).to.equal('0.001');
```

Comment:
Are these the super small number cases? I was assuming it would be more a number like .000004 or something like that. Just curious to understand.